google/resource_container_cluster.go

```diff
@@ -167,6 +167,38 @@ func resourceContainerCluster() *schema.Resource {
 				Description: `The list of zones in which the cluster's nodes are located. Nodes must be in the region of their regional cluster or in the same region as their cluster's zone for zonal clusters. If this is specified for a zonal cluster, omit the cluster's zone.`,
 			},
 
+			"notification_config": {
```

Comment:
Because this is technically an object and not a list, we will want to add `MaxItems: 1` to this block.